rasa/train.py

```diff
@@ -683,20 +697,37 @@ async def _train_nlu_with_validated_data(
         return _train_path
 
 
-def _nlu_model_for_finetuning(
+async def _nlu_model_for_finetuning(
     model_to_finetune: Text,
-    new_config: Dict[Text, Any],
+    file_importer: TrainingDataImporter,
     finetuning_epoch_fraction: float = 1.0,
+    also_train_core: bool = False,
 ) -> Optional[Interpreter]:
+
     path_to_archive = model.get_model_for_finetuning(model_to_finetune)
     if not path_to_archive:
         return None
 
+    rasa.shared.utils.cli.print_info(
+        f"Loading NLU model from {path_to_archive} for finetuning...",
+    )
     with model.unpack_model(path_to_archive) as unpacked:
         _, old_nlu = model.get_model_subdirectories(unpacked)
+        new_fingerprint = await model.model_fingerprint(file_importer)
```

Comment:
mhm, considering that we already called this before (to check if NLU needs to be trained) caching might be an idea 🤔 (out of the scope of this PR)